<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 99, 169);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Sales Table</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>871</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 136, 211);</string>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SoldProducts</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Subtotal</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PaymentType</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Was the order billed?</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>892</x>
     <y>680</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(156, 208, 93);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Back to Reports</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
